vendor/github.com/containerd/containerd/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/containerd/containerd
 go 1.16
 
 require (
-	github.com/Microsoft/go-winio v0.4.17
+	github.com/Microsoft/go-winio v0.5.0
```

Comment:
Haven't checked yet, but any other (indirect) dependency from containerd that needs updating?

Reply:
Yes we are already on 0.5.1: https://github.com/moby/moby/blob/229bc1ee8bd3939634a5eb3bfebfd59aed399e2d/vendor.conf#L3